Voycar.Api.Web.Tests.Integration/Setup/ClientFactory.cs

```diff
@@ -136,9 +140,7 @@ public static async Task<HttpClient> CreateAdminClient(AppFixture<Program> app,
         // Create user entity
         var user = new User()
         {
-            Email = email,
-            PasswordHash = BCrypt.Net.BCrypt.EnhancedHashPassword(password),
-            RoleId = roleId
+            Email = email, PasswordHash = BCrypt.Net.BCrypt.EnhancedHashPassword(password), RoleId = roleId
```

Comment:
Wenn wir das überall machen, dann sollte das hier auch hin
```suggestion
            Email = email.ToLowerInvariant(), PasswordHash = BCrypt.Net.BCrypt.EnhancedHashPassword(password), RoleId = roleId
```